inji-vc-renderer-mock-service/index.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const app = express();
+const port = 3000;
```

Comment:
don't use 3000 as it's very common for UI components

Reply:
Updated to 3008